_i18n/en.yml

```diff
@@ -0,0 +1,81 @@
+---
+global:
+  brazilian portuguese: Português
+  english: English
+header:
+  stories: Stories
+  explore: Explore
+  about: About
+support:
+  fund_us: Support the project
+home:
+  about: Artificial Intelligence for social control of public administration
+  about_rosie: Operação Serenata de Amor created a Robot - An artificial intelligence capable of analysing each request for reimbursement from congresspeople and identify it’s illegal probability.
+  last_histories: Last published histories
+  loading: Loading...
+  more_posts: More posts
+footer:
+  about: about
+  github: Serenata on github
+  contact: contact
+  partners: partners
+stories:
+  published: Published stories
+  loading: Loading...
+explore:
+  reports: Reports
+  about_reports: Rosie, our Robot, in numbers
+  tweet_about: Tweet about this number
+  chamber_reports: Reports done to the Chamber of Deputies
+  suspitions: Different congresspeople reported for suspicious reimbursements
+  questionated: thousand Brazilian Reals in suspicious expenses questioned
+  paragraph_one: Those values correspond to 371% of what we raised with our crowdfunding campaign (~U$ 25.000,00)
```

Comment:
* _This amount corresponds_ (instead of _Those values correspond_)
* _through our crowdfunding_ (instead of _with our crowdfunding_)
*  _25,000.00_ (in English commas are thousand separators and periods are decimal separators)
* _25,000.00 USD_ **or** _US$ 25,000.00_ (instead of _U$_)